File: OOAD/shopping-cart-app/src/com/techlabs/model/Customer.java
package com.techlabs.model;

import java.util.ArrayList;
import java.util.Iterator;
import java.util.List;
import java.util.UUID;

public class Customer {

	private UUID id;
	private String name;
	private String address;
	private List<Order> orders = new ArrayList<Order>();

	public Customer(String id, String name, String address) {
		this.id = UUID.fromString(id);
		this.name = name;
		this.address = address;
	}

	public void addOrder(Order order) {
		orders.add(order);
	}

	public UUID getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public String getAddress() {
		return address;
	}

	public List<Order> getOrders() {
		return orders;
	}
}

File: typescript/customer/customer.ts
export class Customer{
    constructor (private _id :number ,private _name :String){}
      public getId():number {
           return this._id;
      }
      public getName() :String{
          return this._name;
      }
     
   }
   
   export class Address{
       constructor(private _address :String){}
       public getAddress():String{
           return this._address;
       }
   
   }
   
  